### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by wendig0x** [simplex-chat/simplex-chat#4737]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android

### OS version

13

### App version

6.0.1

### Current Behavior

If I delete all SMP servers and add my own (only one server is shown in the list), there are still two servers active in the “Server Information” tab - my own and smp15.simplex.im. Why?

### Expected Behavior

There should only be one connection - to my server

### Steps To Reproduce

1. Add your own smp server
2. Delete all others
3. Delete all chats and recreate them with the new server

**Comment by wendig0x** [comment]
Yes, you are right. I double-checked, now everything works correctly.

In order not to create another topic, I will ask here: does a private smp-server somehow connect to other servers in the SimpleX network by itself?

**Comment by Skybladev2** [comment]
I also noticed that Simplex client tries to connect to 2 additional default simplex servers (smpX.simplex.im), despite I turned all of them off and use only single private server.
SimpleX address is from the private server. When I check the contact sending and receiving servers they both configured to the same private server.
As a result, the client tries to send a message through default servers, which are unavailable in my network, times out and then eventually sends the message using the correct server. The message is delayed by a minute which is suboptimal.

How can I remove all unneeded servers from the server list? I can see their statistics only.

**UPD**: when I deleted all chats and contacts from default servers messages started to be sent instantly.
